### ISSUE TITLE
Is the CIFAR10 dataset error rate given in the paper the result of a single model?

### ISSUE THREAD

**Issue by lyl120117** [wy1iu/LargeMargin_Softmax_Loss#33]
@wy1iu I tried to reproduce L-Softmax with pytorch, the CIFAR10 dataset best error rate is 10.5 when m=2, higher than 7.73 in the paper, I would like to know if the 7.73 error rate is the result of a single model, or if multiple models are integrated?
I am looking forward to your reply, thanks.
